updates to remove vernum ref in cloud doc instances (#467)
opendatahub-io/opendatahub-documentation: modules/enabling-data-science-pipelines-2.adoc

====
After you upgrade to {productname-short} 2.9 or later, pipelines created with data science pipelines 1.0 continue to run, but are inaccessible from the {productname-short} dashboard. If you are a current data science pipelines user, do not upgrade to {productname-short} with data science pipelines 2.0 until you are ready to migrate to the new pipelines solution.
====
ifdef::self-managed[]
To upgrade to {productname-short} {vernum} with data science pipelines 2.0, ensure that there is no installation of Argo Workflows that is not installed by data science pipelines on your cluster, and follow the upgrade steps described in link:{odhdocshome}/upgrading-open-data-hub/[Upgrading {productname-short}].

If you upgrade to {productname-short} {vernum} with data science pipelines enabled and an Argo Workflows installation that is not installed by data science pipelines exists on your cluster, {productname-short} components will not be upgraded. To complete the component upgrade, disable data science pipelines or remove the separate installation of Argo Workflows. The component upgrade will complete automatically.
endif::[]
ifdef::cloud-service[]

> To upgrade to {productname-short} with data science pipelines 2.0, ensure that there is no installation of Argo Workflows that is not installed by data science pipelines on your cluster, and follow the upgrade steps described in link:{odhdocshome}/upgrading-open-data-hub/[Upgrading {productname-short}].

Reviewer: ```suggestion
To upgrade to {productname-short} with data science pipelines 2.0, ensure that data science pipelines on your cluster have not installed Argo Workflows, and follow the upgrade steps described in link:{odhdocshome}/upgrading-open-data-hub/[Upgrading {productname-short}].
```

Trying to cut down on the amount of negatives in the sentence for clarity - but may have obscured the meaning. Feel free to rework or revert if this doesn't make sense!

Reviewer 2: The point being conveyed here is that the component "data science pipelines" automatically installs Argo Workflows. Customers might have installations of Argo workflows that predate their implementation of data science pipelines and RHOAI. These external installations of Argo workflows could conflict and cause problems with the version of Argo that is installed by data science pipelines. Because of this, the suggested change is not correct.
